### PR TITLE
Automatic commit: Move cp-starter-digitalexp-mobilecards-1-setup, cp-…

### PR DESCRIPTION
**Pull request by jitendrakansal**
…starter scenario for sap mobile cards

### CODE REVIEW

File: tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+On the SAP Cloud Platform Trial landing page, choose **Enter your Trial Account**.
+
+![MobileCards](img_0.png)
+
+>The trial account should contain one `subaccount` and `space`.
+
+Choose **trial**.
+
+![cf Trial](img_2.png)
```
Comment (GevorgyanV) [inline tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md]: this photo in test description is not confidential enought. space name and region should be blurred.

File: tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+You can now view both the cards.
+
+![MobileCards](img_11.jpg)
+
+[OPTION END]
+
+[OPTION BEGIN [iOS]]
+
+Scan the QR code with your camera and select the toast message to launch the SAP Mobile Cards app.
+
```
Comment (GevorgyanV) [inline tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md]: QR-code is incorrect, 
for iOS not working tool, 
for Android opens API L**** Apotheke

File: tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+You can find more details on the [SAP Mobile Cards developer page](https://developers.sap.com/topics/mobile-cards.html).
+
+![MobileCards](MobileCards.png)
+
+[DONE]
+[ACCORDION-END]
+[ACCORDION-BEGIN [Step 2: ](Navigate to the SAP Mobile Services cockpit)]
+
+On the SAP Cloud Platform Trial landing page, choose **Enter your Trial Account**.
```
[the remark this answers is not in the document]
Comment (jitendrakansal) [reply tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md]: @GevorgyanV direct link added

[the remark this answers is not in the document]
Comment (jitendrakansal) [reply tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md]: @GevorgyanV which account? For this tutorial, assumption is user is already on-boarded to CF trial account.

File: tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md (hunk 1 above)
Comment (jitendrakansal) [reply tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md]: Done

File: tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md (hunk 2 above)
Comment (jitendrakansal) [reply tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md]: Are you pointing to QR code from tutorial? If so, i have blurred it. 
You should be able to scan the QR code from your account cockpit.

**Comment by jitendrakansal** [comment]
updated with @GevorgyanV feedback.

File: tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
+---
+title: Configure Initial Setup for SAP Mobile Cards
+description: Define a destination in SAP Cloud Platform Mobile Services cockpit to establish connectivity to a SAP SuccessFactors sandbox API.
+auto_validation: true
+primary_tag: products>sap-mobile-cards
+tags: [ tutorial>beginner, operating-system>ios, operating-system>android, topic>mobile, products>sap-cloud-platform, products>sap-mobile-cards, software-product-function>sap-cloud-platform-mobile-services ]
```
Comment (thecodester) [inline tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md]: OK

File: tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md (hunk 2 above)
Comment (GevorgyanV) [reply tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md]: > Are you pointing to QR code from tutorial? If so, i have blurred it.
> You should be able to scan the QR code from your account cockpit.

no, I scanned from the cockpit

Comment (jitendrakansal) [reply tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md]: Did you already install Mobile Cards client on your device? Are you trying on iOS or Android?

**Pull request merged**